core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -48,6 +48,27 @@ private CatalogProperties() {}
 
   public static final boolean CACHE_CASE_SENSITIVE_DEFAULT = true;
 
+  /**
+   * Controls policy of caffeine cache
+   *
+   * <p>Behavior of specific values of cache.strategy:
+   *
+   * <ul>
+   *   <li>EXPIRE_AFTER_ACCESS - cache entries are never evicted as long as they are being accessed
+   *       frequently
```

Comment:
This definition seems vague somewhat. Frequently is not very specific. Why not flip the definition and say `cache entry is removed from the cache after a specified time has passed since its last access or write operation`